source/includes/_holding_totals.md

```diff
@@ -0,0 +1,61 @@
+# Holding Totals
+
+## GET > Retrieve holding totals for an account
+
+```shell
+curl "https://app.sixpark.com.au/api/v1/users/:user_id/accounts/:account_id/portfolio/holding_totals"
+  --header "Accept: application/json"
+  --header "Authorization: Bearer <access_token>"
+
+```
+
+> A successful (200 HTTP status code) example JSON response body:
+
+```json
+{
+  "data": [
+    {
+      "id": "af66cddf-fc5b-4fd8-badf-99c71c4e6d69",
+      "type": "holdings_total",
+      "attributes": {
+        "as_at_date": "2020-09-06",
+        "account_value": 1234567.89
+      },
+      "relationships": {
+        "portfolio": {
+          "data": {
+            "id": "5326fc80-f144-44c6-839b-07f4a911de12",
+            "type": "portfolio"
+          }
+        }
+      }
+    },
+    "..."
+  ]
+}
+```
+
+_Retrieve_ a collection of the resource owner's accounts.
+
+### HTTP Request
+
+`GET https://app.sixpark.com.au/api/v1/users/:user_id/accounts/:account_id/portfolio/holdings_totals`
+
+### Parameters
+
+Not Applicable
+
+### Summary
+
+Configuration | Value | Description
+--------- | ------- | -----------
+authenticated | resource owner | Access is granted via the resource owner's access token
+paginated | no |
+
+### 201 HTTP status code properties
+
+Property | Type | Description
+--------- | ----------- | -----------
+id | string | Unique identifier for the holdings total object
+as_at_date | string | The date represented by this snapshot of the acounts performance
```

Comment:
We should settle on either `performance`, `holding_totals` or `value of the account` and remain consistent throughout.